editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1242,6 +1242,7 @@ bool CanvasItemEditor::_gui_input_zoom_or_pan(const Ref<InputEvent> &p_event, bo
 	}
 
 	if (panner_active) {
+		_update_cursor();
```

Comment:
I believe, that a better location to fix the linked bug is to move `_update_cursor();` next to  `pan_pressed = panner->is_panning();` a few lines above.

Reason is that `_update_cursor()` should only be called when the pan-status changes and not on every mouse-move event.

Reply:
Good catch. Thx
I've pushed an update.
But I did a rebase, and it seems to have messed a lot of files, and woke up a bunch of reviewers.
It doesn't seem right.